mmv1/third_party/terraform/website/docs/d/apphub_discovered_workload.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+subcategory: "Apphub"
+description: |-
+  Get information about a discovered workload.
+---
+
+# google\_apphub\_discovered_workload
+
+Get information about a discovered workload from its uri.
+
+
+## Example Usage
+
+
+```hcl
+data "google_apphub_discovered_workload" "my-workload" {
+  location = "my-location"
+  workload_uri = "my-workload-uri"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - The project where we are looking up the discovered workload from
+* `workload_uri` - (Required) The uri of the workload.
+* `location` - (Required) The location you want to get the discovered worklaod from
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are exported:
+
+* `discovered_workload` - Represents a network/api interface that exposes some functionality to clients for consumption over the network. Structure is [documented below](#nested_discovered_workloads)
+
+<a name="nested_discovered_workloads"></a>A `discovered_workload` object would contain the following fields:-
+
+* `name` - Resource name of a Workload. Format: "projects/{host-project-id}/locations/{location}/applications/{application-id}/workloads/{workload-id}".
```

Comment:
Does the format include `applications/{application-id}`?

Reply:
Yes, the response if of that format.